plugins/modules/katello_content_view.py

```diff
@@ -25,20 +25,23 @@
     - Create and Manage Katello content views
 author: "Eric D Helms (@ehelms)"
 requirements:
-    - "nailgun >= 0.28.0"
+    - apypie
```

Comment:
Not just a refactor, I guess?

Reply:
Well it is refactoring in that the change of used library should be invisible to the user.
But yes this is primarily a conversion from Nailgun to apypie.